backend/src/test/java/org/backend/ridinglessons/RidinglessonServiceTest.java

```diff
@@ -0,0 +1,31 @@
+package org.backend.ridinglessons;
+
+import org.backend.ridinglesson.Ridinglesson;
+import org.backend.ridinglesson.RidinglessonRepo;
+import org.backend.ridinglesson.RidinglessonService;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+class RidinglessonServiceTest {
+    RidinglessonRepo ridinglessonRepo = mock(RidinglessonRepo.class);
+    RidinglessonService ridinglessonService = new RidinglessonService(ridinglessonRepo);
+
+    @Test
+    void findAllLessons(){
+        //Given
+        Ridinglesson ridinglesson1 = new Ridinglesson("1", "Kenzie", "Libertydressage", "Lui", "02.03.2024","15:00");
+        Ridinglesson ridinglesson2 = new Ridinglesson("2", "Jessy", "Dressage", "Dalera", "03.03.2024","15:00");
+        List<Ridinglesson> ridinglesson = List.of(ridinglesson1, ridinglesson2);
```

Comment:
da es sich hier um eine Liste handelt, ist es besser, dass man das auch im Variablennamen sieht, also ridingLessons (plural) statt ridingLesson (singular)
```suggestion
        List<Ridinglesson> ridingLessons = List.of(ridinglesson1, ridinglesson2);
```